bin/nib-ls

```diff
@@ -22,6 +22,7 @@ import sys
 from math import ceil
 from optparse import OptionParser, Option
 from io import StringIO
+from collections import Counter
```

Comment:
I don't mind reimplementing without Counter if necessary. Myself stopped carrying about 2.6 about a year ago

On April 6, 2016 5:36:33 PM EDT, Matthew Brett <notifications@github.com> wrote:
>> @@ -22,6 +22,7 @@ import sys
>>  from math import ceil
>>  from optparse import OptionParser, Option
>>  from io import StringIO
>> +from collections import Counter
>
>Not compatible with Python 2.6.  We can drop Python 2.6 unless there's
>an obvious workaround.
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/nipy/nibabel/pull/437/files/0851f93c446db78ce7515bb5966ffa91dd043ba4#r58786429